modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/conversion/ConversionUtils.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2005-2014 Restlet
+ * 
+ * The contents of this file are subject to the terms of one of the following
+ * open source licenses: Apache 2.0 or or EPL 1.0 (the "Licenses"). You can
+ * select the license that you prefer but you may not use this file except in
+ * compliance with one of these Licenses.
+ * 
+ * You can obtain a copy of the Apache 2.0 license at
+ * http://www.opensource.org/licenses/apache-2.0
+ * 
+ * You can obtain a copy of the EPL 1.0 license at
+ * http://www.opensource.org/licenses/eclipse-1.0
+ * 
+ * See the Licenses for the specific language governing permissions and
+ * limitations under the Licenses.
+ * 
+ * Alternatively, you can obtain a royalty free commercial license with less
+ * limitations, transferable or non-transferable, directly at
+ * http://restlet.com/products/restlet-framework
+ * 
+ * Restlet is a registered trademark of Restlet S.A.S.
+ */
+
+package org.restlet.ext.apispark.internal.conversion;
+
+import org.restlet.engine.util.StringUtils;
+
+public class ConversionUtils {
+
+    /**
+     * Generates a name for a resource computed from its path. The name is
+     * composed of all alphanumeric characters in camel case.<br/>
+     * Ex: /contacts/{contactId} => ContactsContactId
+     * 
+     * @param uri
+     *            The URI of the Resource
+     * @return The Resource's name computed from the path.
+     */
+    public static String processResourceName(String uri) {
+        String processedUri = "";
```

Comment:
If you need to build a string on several instruction (other that `"a" + "b" + "c"`), use a `StringBuilder`